.travis.yml

```diff
@@ -0,0 +1,48 @@
+language: node_js
```

Comment:
Mind updating this file based on https://github.com/ciena-blueplanet/ember-prop-types/blob/master/.travis.yml